actions/bellsoft-liberica-dependency/main.go

```diff
@@ -169,14 +169,20 @@ func main() {
 	outputs.Write()
 }
 
-func determineNikVersion(r Release, additionalOutputs actions.Outputs) string {
+func determineNikVersion(r Release, additionalOutputs actions.Outputs, isSource bool) string {
 	key := ""
-	if v, err := actions.NormalizeVersion(r.Components[0].Version); err != nil {
+	var componentKey int
```

Comment:
Could you add a comment and indicate what these magic 0 / 2 values indicate? What does the data structure look like presently? If something changes in the API again, it'll be tricky to compare to how things used to be without that context.

Reply:
ah actually; I just noticed something: in both  https://api.bell-sw.com/v1/nik/releases?package-type=src.tar.gz&version-modifier=latest&bundle-type=standard&component-version=liberica%4017 and https://api.bell-sw.com/v1/nik/releases?arch=x86&bitness=64&os=linux&package-type=tar.gz&version-modifier=latest&bundle-type=core&component-version=liberica%4017 I could filter the components with

```
{
        "version": "17.0.10+13",
        "embedded": true,
        "component": "liberica"
      }
```

instead of a magic and probably not reliable at all index number

I'll do that

Reply:
done! please review!